DESCRIPTION

```diff
@@ -41,7 +41,6 @@ Suggests:
     jsonlite,
     mockery,
     patrick,
-    pkgdown,
```

Comment:
Is putting pkgdown in Suggests not the usual way of signaling we use it for development?

Reply:
Nope.

Reply:
You can sample any of the r-lib, tidyverse, etc. organizations packages, all of which use _pkgdown_, but none of them have _pkgdown_ in `Suggests`. As far as CRAN is concerned, it's irrelevant.

Reply:
Okay, thanks for clarifying.